app/views/hub/users/edit_role.html.erb

```diff
@@ -49,7 +49,7 @@
         <div class="form-group">
           <%= label_tag(:organization, t("devise.invitations.new.organization_label"), class: "form-question") %>
           <div class="form-width--long">
-            <% org_value = @user.role.is_a?(OrganizationLeadRole) && @user.role.organization.present? ? [{ value: @user.role.organization.id, name: @user.role.organization.name }].to_json.to_s.html_safe : [] %>
+            <% org_value = @user.role.is_a?(OrganizationLeadRole) && @user.role.organization.present? ? taggable_format([{ value: @user.role.organization.id, name: @user.role.organization.name }]) : [] %>
```

Comment:
Non-blocking - would it improve cleanliness for us to return the same data type no matter which side of the ternary we're on, like above?